app/views/claims/confirmation.html.erb

```diff
@@ -20,16 +20,21 @@
     </p>
 
     <p class="govuk-body">
-      They will verify the information you have provided and then send
-      you the outcome of your application within 4 weeks.
+      They will verify the information that you have provided and then
```

Comment:
Not for this PR, but I do find it weird that we say use "They" and the the line about sending "your application to the Department for Education" when we generate the email and it comes from "Claim additional payments for teaching" with a reply-to address of studentloanteacherpayment@digital.education.gov.uk. Something to chat to Patrick about when he's back?

Reply:
I've already sent Patrick an email with screen shots of this because I agree, its a little weird. He should be back tomorrow, so when I see him next I'll pick it up with him.